src/main/java/org/jboss/tools/intellij/openshift/utils/odo/Odo.java

```diff
@@ -101,6 +101,6 @@
 
     String consoleURL(OpenShiftClient client) throws IOException;
 
-    void debug(String project, String application, String context, String component, Integer port) throws IOException;
+    void debug(String project, String application, String context, Integer port) throws IOException;
```

Comment:
For the uniformity of the API, I would keep this parameter as seems logical to pass it when we are debugging a component (even if the CLI implementation does not use it as Odo will deduce the component name from the path.

Reply:
reverted. see #159 to fix uniformity with the link method.